File: v1/src/main/java/com/api/v1/medical_slots/services/MedicalSlotManagementService.java
package com.api.v1.medical_slots.services;

import org.springframework.http.ResponseEntity;

public interface MedicalSlotManagementService {
    ResponseEntity<Void> cancel(String medicalLicenseNumber, String id);
    ResponseEntity<Void> complete(String medicalLicenseNumber, String id);
}

File: v2/src/main/kotlin/com/api/v2/medical_slots/services/MedicalSlotManagementService.kt
package com.api.v2.medical_slots.services

import org.springframework.http.ResponseEntity

interface MedicalSlotManagementService {
    fun cancel(medicalLicenseNumber: String, slotId: String): ResponseEntity<Unit>
    fun complete(medicalLicenseNumber: String, slotId: String): ResponseEntity<Unit>
}